api-tests/specs/role-service/roleservice.js

```diff
@@ -1,39 +1,36 @@
 const requests = require('../../data/role-service/roleservice')
 const conf = require('../../config/conf')
-
-beforeEach(function () {
-    baseRequest = request(conf.BASE_URL)
-})
+const endpointUri = ':82/roles';
```

Comment:
is there a way of not hard-wiring the port number?

Reply:
Left that as an exercise for QA-  have chatted to Sameera about it. Can remove these when we get the gateway in.